src/libultra/io/contramwrite.c

```diff
@@ -1,3 +1,68 @@
+#include "ultra64.h"
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/contramwrite/__osContRamWrite.s")
+s32 __osContRamWrite(OSMesgQueue* mq, s32 channel, u16 address, u8* buffer, s32 force) {
```

Comment:
Same as `__osContRamRead`, shouldn't be better `void*` instead of `u8*`?
Also, here that parameter is called `buffer` while `__osContRamRead` has `data`. Wouldn't be better to use the same name for both?